gradle/libs.versions.toml

```
@@ -1,7 +1,8 @@
 [versions]
-agp = "8.12.0-alpha06"
+agp = "8.11.0"
```

p1: 빌드이슈로 agp 버전 다운 희망합니다~
```suggestion
agp = "8.10.1"
```